memory-game.js

```diff
@@ -117,12 +118,10 @@ H5P.MemoryGame = (function (EventDispatcher, $) {
       $taskComplete.show();
       $feedback.addClass('h5p-show'); // Announce
       $bottom.focus();
+      score = 1;
 
       // Create and trigger xAPI event 'completed'
-      var completedEvent = self.createXAPIEventTemplate('completed');
-      completedEvent.setScoredResult(1, 1, self, true, true);
-      completedEvent.data.statement.result.duration = 'PT' + (Math.round(timer.getTime() / 10) / 100) + 'S';
```

Comment:
Makes total sense to change this based on what the xAPI specification says! Hopefully, not too many people have been using this to keep track of the time the user took to complete the game. So, the only thing that I have to add as a suggestion is to add an xAPI extension to the `property` (see https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Data.md#245-result - I think http://id.tincanapi.com/extension/duration would fit) that holds the time the user took to complete the game as computed before. This way, you follow the xAPI specification but also allow others to retain the time spent.

Reply:
I think with the addition of `this.setActivityStarted()`, the duration property will be set in [`H5P.XAPIEvent.prototype.setScoredResult`](https://github.com/h5p/h5p-php-library/blob/f6fef74651264dfad4c22c290d730ae96b7ee20a/js/h5p-x-api-event.js#L59), so the final statement should be the same as before.

Reply:
Am I wrong here? If you rely on `this.setActivityStarted()`, then you'll get the time that has passed since the content was attached to the DOM. In the current implementation, however, the duration is set to the time that has passed since the timer has been started when first clicking on a card.

Reply:
Thank you for pointing this out! It is not our intention to change this. I will remove the call to `this.setActivityStarted();` that was added and bring back the previous statement for setting the duration.

Reply:
Well, I think it would in fact make sense to change it as mentioned (as that's what the `duration` property is meant to mean in xAPI) and that's what all other H5P content types use it for :-) I was merely suggesting to also offer those who currently might use that value to track the time used (don't know if anyone actually uses that at all) a way to still obtain the "elapsed time", and that could go into an extension.

Reply:
Your insight is much appreciated, thank you. Changing the way the duration is calculated was outside the scope of the task being worked on. We will try to keep this in mind if we decide to address this in the future.